.php_cs.dist

```diff
@@ -10,7 +10,7 @@ with this source code in the file LICENSE.
 EOF;
 
 $finder = PhpCsFixer\Finder::create()
-    ->exclude(['var/cache', 'tests/Resources/cache'])
+    ->exclude(['var/cache', 'tests/Resources/cache', 'node_modules'])
```

Comment:
php-cs-fixer always tries to fix some php files in `node_modules/flatted`

Reply:
I guess this should be fixed in 2.1, so I'll create an additional pull request ... Didn't think about that before